relay-general/src/processor/selector.rs

```diff
@@ -47,11 +47,12 @@ pub enum SelectorPathItem {
     Key(String),
     Wildcard,
     DeepWildcard,
+    Other(String),
 }
 
 impl fmt::Display for SelectorPathItem {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match *self {
+        match self {
```

Comment:
im getting borrow check issues if i try to display a string when matching on '*self'